DependencyInjection/CmfSeoExtension.php

```diff
@@ -290,4 +296,29 @@ private function loadSitemapHandling($config, XmlFileLoader $loader, ContainerBu
             $container->removeDefinition($this->getAlias().'.sitemap.guesser.alternate_locales');
         }
     }
+
+    /**
+     * Configuration block to configure form.
+     *
+     * @param array            $config
+     * @param ContainerBuilder $container
+     */
+    private function loadFormConfiguration($config, ContainerBuilder $container)
+    {
+        $seoMetadataClass = null;
+        if (isset($config['data_class'])
+            && isset($config['data_class']['seo_metadata'])
+            && null !== $config['data_class']['seo_metadata']
+        ) {
+            $seoMetadataClass = $config['data_class']['seo_metadata'];
+        } else {
+            if ($container->hasParameter($this->getAlias() . '.backend_type_phpcr')) {
```

Comment:
yes, its for the mapping pass to know if it actually should map. this is because we need to register compiler passes before we know the configuration, so they run unconditionally and hae to check themselves if they are needed.